pkgs/top-level/python-packages.nix

```diff
@@ -23203,6 +23203,28 @@ in modules // {
     };
   };
 
+  you-get = buildPythonPackage rec {
+    version = "0.4.390";
+    name = "you-get-${version}";
+    namePrefix = "";
+    disabled = !isPy3k;
+
+    # Tests aren't packaged, but they all hit the real network so
+    # probably aren't suitable for a build environment anyway.
+    doCheck = false;
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/y/you-get/${name}.tar.gz";
+      sha256 = "17hs0g9yvgvkmr7p1cz39mbbvb40q65qkc31j3ixc2f873gahagw";
+    };
+
+    meta = {
+      description = "a tiny command line utility to download media contents from the web";
```

Comment:
Please capitalize the description, per the guidelines. Also, I suggest that you add yourself as the maintainer; adding orphaned packages doesn't make too much sense.